conanfile.py

```diff
@@ -49,13 +48,34 @@ def generate(self):
         tc.generate()
 
     def build(self):
+        if os.environ.get("BUILD_DIR_FULL_PATH") is not None:
+            return
```

Comment:
Do we have documentation for all of the variables (both environment and command line) that are taken by the build scripts? This should be added there with an explanation of what it is, what it does, and how to set it.